### PR TITLE
Comment option in verification dialog

### PR DESCRIPTION
**Pull request by rush42**
Do not merge!
This PR updates the API calls for the verification procedure so they support the newly added comment field.

### CODE REVIEW

**Comment by rush42** [comment]
[Backend PR](https://github.com/FixMyBerlin/atlas-geo/pull/31)

File: src/api/api.ts [hunk 1]
```diff
@@ -21,6 +21,7 @@ export type VerificationApiPost = {
   verified_at: string
   verified_by: number | undefined
   verified_status: VerificationStatus | null
+  comment: string | undefined
```
Comment (rush42) [inline src/api/api.ts:24]: not sure how we want to handle nullability

File: src/api/api.ts [hunk 2]
```diff
@@ -48,6 +50,7 @@ export const updateVerificationStatus = ({
   encoded.append('verified_at', verified_at)
   encoded.append('verified_by', verified_by.toString())
   encoded.append('verified_status', verified_status)
+  encoded.append('comment', comment || '')
```
Comment (rush42) [inline src/api/api.ts:53]: see comment above about nullability

**Comment by tordans** [comment]
This was manually merged. Thanks!

**Pull request closed without merging**